mteb/tasks/Classification/eng/YahooAnswersTopicsClassification.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskClassification
+
+
+class YahooAnswersTopicsClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="YahooAnswersTopicsClassification",
+        description="Dataset composed of questions and answers from Yahoo Answers, categorized into topics.",
+        reference="https://huggingface.co/datasets/yahoo_answers_topics",
+        dataset={
+            "path": "yahoo_answers_topics",
+            "revision": "78fccffa043240c80e17a6b1da724f5a1057e8e5",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="accuracy",
+        date=None,
+        form=None,
+        domains=["Web"],
+        task_subtypes=None,
+        license=None,
+        socioeconomic_status=None,
+        annotations_creators=None,
+        dialect=None,
+        text_creation=None,
+        bibtex_citation=None,
+        n_samples={"train": 1400000, "test": 60000},
```

Comment:
```suggestion
        n_samples={"test": 60000},
```
No need for the train metadata since `eval_splits` only contains test